spatialos/UnityGDK
# UTY-1215: entity builder helpers in feature modules

**Contributions**: We are not currently taking public contributions - see our [contributions](CONTRIBUTING.md) policy. However, we are accepting issues and we do want your [feedback](../README.md#give-us-feedback).

-------

#### Description
Added helper methods to add feature module-specific components to an entity builder - so users do not have to worry about what components each feature module may use. Will also check if the snapshot is valid and if playground isn't broken. Also combined SystemHelper classes with EntityBuilderHelper, unifying helper methods into one class per feature module. 
#### Tests
- check snapshot sanity
- check worker/client in playground
#### Documentation
none
#### Primary reviewers
If your change will take a long time to review, you can name at most two primary reviewers who are ultimately responsible for reviewing this request. @ mention them.
